static/app/views/explore/tables/aggregatesTable.tsx

```diff
@@ -54,10 +54,18 @@ export function AggregatesTable({}: AggregatesTableProps) {
   const [query] = useUserQuery();
 
   const eventView = useMemo(() => {
+    // We need the three fields to be able to scroll to a specific span in the trace view
+    const eventViewFields = [...fields];
+    if (fields.includes('span_id') || fields.includes('id')) {
+      eventViewFields.push('transaction.span_id');
+      eventViewFields.push('timestamp');
+      eventViewFields.push('trace');
+    }
```

Comment:
Is this if the user does a group by on the span id? If so, let's leave this out for now as this affects the actual query being made and we should think about how grouping on ids should work in explore.

Reply:
Yeah, it is for groupby `span_id`. I'll remove it